projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -235,6 +249,14 @@ export class ExplorerComponent {
   }
 
   private mapToInitialState(param: ParamMap): InitialExplorerState {
+    const series: ExploreSeries[] = param
```

Comment:
This could be reverted right? it's no longer being used by the order by decode. 

That said, do we handle the default order by from series case too? I don't see that logic moved anywhere else, just removed (I very well may have missed it though).

Basically, I would expect to be able to do all of
- Change interval to none, and have a default order by added based on my current first series (no order by in the url to start, but added after)
- Every time I change the ordering details, the URL is updated appropriately.
- At any time, I can grab the url and reload the page (or in another tab) with the order by I had selected being restored.

Reply:
I should remove the series and orderBy const and assign them to the underneath object. These 3 scenarios are currently satisfied.

Reply:
Can you point me to the code that does the first one?

Reply:
You are right. Currently, I'm taking undefined as None. When there is no URL param for order, the attribute will be taken as None and the order by will not be applied. I suppose that I will need a new state there, the NONE state, the undefined state and the regular.

Reply:
We don't actually want a non state though - if the url param isn't defined, we should default to the value of the first series metric desc. there should be no way to get to a none state (other than setting an interval, in which case the order by won't be visible)

Reply:
I already added a None state (in attribute) to prevent the re-ordering. Do you say that I need to remove it? If the user clicks on None, the order is not applied.

Reply:
I think the idea had been to remove None, correct. There's no value in having it - everything has some order, so let's pick a reasonable default that the user can change. There's no use case that I can think of where a user would say - "I want random order".

Reply:
done all scenarios